.gitignore

```diff
@@ -3,3 +3,5 @@ npm-debug.log
 build
 .antwar
 
+.idea
```

Comment:
As far as I know everyone is putting this things into the projects. But if this is a problem I can add it to my global `.gitignore `not a problem